MijnApp-Frontend-React/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "eidhoven",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.5.0",
+    "@testing-library/user-event": "^7.2.1",
+    "axios": "^0.19.2",
+    "material-ui": "^0.20.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-redux": "^7.2.1",
```

Comment:
I suggest you to add the redux-toolkit library. 
Because It consist of a lot of useful functions for redux
https://redux-toolkit.js.org/introduction/quick-start